python/paddle/fluid/tests/unittests/ir/inference/test_trt_reshape2_matmul_fuse_pass.py

```diff
@@ -12,153 +12,112 @@
-from functools import partial
-from typing import Optional, List, Callable, Dict, Any, Set
-import unittest
+from __future__ import print_function
 
-import hypothesis
-from hypothesis import given, settings, seed, example, assume, reproduce_failure
-import hypothesis.strategies as st
-
-
-class TestReshape2MatmulFusePass(PassAutoScanTest):
-    """
-        x_var
-          |
-       reshape2
-          \
-    reshape2_out_var    y_var
-             \           /
-                 matmul      bias_var
-                    \          /
-                   elementwise_add
-    """
-
-    def sample_predictor_configs(self, program_config):
-        # TRT
-        config = self.create_trt_inference_config()
-        config.enable_tensorrt_engine(
-            max_batch_size=10,
-            workspace_size=102400,
-            min_subgraph_size=0,
-            precision_mode=paddle_infer.PrecisionType.Float32,
-            use_static=False,
-            use_calib_mode=False)
-        yield config, ['mul', 'elementwise_add'], (1e-4, 1e-1)
-
-    def add_ignore_pass_case(self):
-        # Here we put some skip rules to avoid known bugs
-        def teller1(program_config, predictor_config):
-            y_shape = list(program_config.weights["matmul_y"].shape)
-            bias_shape = program_config.weights["bias"].shape
-            axis = program_config.ops[2].attrs["axis"]
-            # bias should be [mul_y_shape[-1]]
-            if axis == 0 or bias_shape[0] != y_shape[1] or len(bias_shape) != 1:
-                return True
-            return False
-
-        self.add_ignore_check_case(
-            teller1,
-            IgnoreReasons.PASS_ACCURACY_ERROR,
-            "The pass error on TRT while shape of bias is not [out_size].",
-        )
-
-    def sample_program_config(self, draw):
-        # 1. Generate shape and attr of reshape2
-        reshape = draw(
-            st.lists(st.integers(min_value=1, max_value=10),
-                     min_size=2,
-                     max_size=2))
-        x_shape = reshape + [1, 1]
-
-        # 2. Generate attr:transpose_X/transpose_Y/alpha of matmul
-        alpha = 1.0
-        transpose_X = False
-        transpose_Y = False
-
-        # 3. Generate legal shape of input:Y of matmul
-        y_shape = draw(
-            st.lists(st.integers(min_value=1, max_value=8),
-                     min_size=2,
-                     max_size=2))
-        y_shape[0] = x_shape[1]
-
-        # 4. Generate legal attr:axis of elementwise_add
-        axis = draw(st.integers(min_value=-1, max_value=1))
-        if axis == 0:
-            axis = -1
-        bias_shape = [
-            y_shape[1],
+import unittest
+import numpy as np
+from inference_pass_test import InferencePassTest
+import paddle.fluid as fluid
+import paddle.fluid.core as core
+from paddle.fluid.core import PassVersionChecker
+from paddle.fluid.core import AnalysisConfig
+
+
+class TRTRoiAlignTest(InferencePassTest):
```

Comment:
1.  Describe the why delete this Pass and what is the alternative solution in this PR.
2.  The name “TRTRoiAlignTest” is inconsistent with file name.

Reply:
> * Describe the why delete this Pass and what is the alternative solution in this PR.
> * The name “TRTRoiAlignTest” is inconsistent with file name.

- this pass is useless,  because reshape2 and matmul is already supported in Paddle-TRT.
- the .py file was just modified to avoiding CI test trt_reshape2_matmul_fuse_pass, later I will remove this .py.